test/test_unary_ufuncs.py

```diff
@@ -0,0 +1,75 @@
+import torch
+
+from torch.testing._internal.common_utils import \
+    (TestCase, run_tests)
+from torch.testing._internal.common_methods_invocations import \
+    (unary_ufuncs)
+from torch.testing._internal.common_device_type import \
+    (instantiate_device_type_tests, ops, onlyOnCPUAndCUDA)
+
+
+# Returns a tensor of the requested shape, dtype, and device
+# Initialization uses randint for non-float types and randn for float types.
```

Comment:
Is this unary ufunc specific?

Reply:
This is the tensor construction helper used by the test I copied. It is not strictly unary ufunc specific, no. I actually have a new generator which I want to replace this with for the unary_ufuncs that I plan to put in to a common file.

For this PR, I would suggest that before it's merged this generator and the copied test are just removed.

Reply:
This has been removed from the PR.